share/completions/apm.fish

```diff
@@ -40,7 +36,11 @@ function __fish_apm_using_command
                         return 1
                     end
                 case '*'
-                    return 1
+                    if test $command_seen_once -eq 0
+                        continue
```

Comment:
That would mean there can't be a package called "stars", "install", "docs", "home" or "open".

And the "benefit" you mention is that fish won't complete - it's not hooked up to syntax highlighting (that would require using `complete`s "-A" option) and doesn't allow the user to undo the error. It's not really a good experience if I'm mashing the TAB key and all I get is that limp blink once. Or even worse, if it decides to complete files again.

Anyway, this is your completion - I don't use apm, so I don't have a stake in this (apart from wanting fish to be great) or expert knowledge of the tool. If you insist, we can keep it that way.